.github/scripts/sync_labels.py

```diff
@@ -48,12 +48,14 @@ def get_linked_pr_from_issue_number(repo, number):
 
 def get_linked_issues_based_on_pr_body(repo, number):
     pr = repo.get_pull(number)
-    pattern = fr'Fixes:? (?:#|{repo}#|https://github.com/{repo}/issues/)(\d+)'
-    matches = re.findall(pattern, pr.body)
+    pattern = fr'(fix(?:|es|ed)|resolve(?:|d|s)|close(?:|d|s))?(?:#|{repo}?:#(\d+)|https.*?/issues/)(\d+)'
```

Comment:
nit: `fix(?:|es|ed)?` since `fix` is valid too
same for `resolve(?:|d|s)?`
we could just list all the available options without optimizing the regex and let re.compile do the optimization job.
then there should be `\s*[: ]\s*` in between the keyword and the issue number.
and only then the issue number number description.

For that, please see see https://github.com/scylladb/scylla-dtest/blob/b168b11055634daf8fd759232e11f051ddf93d4f/tools/marks.py#L164

Reply:
Changed, and removed the `close|s|d`

Reply:
why did you remove close|s|d?

Reply:
Because we add it when we close the pull request , so it's not referring to an issue that we are fixing

Reply:
There were cases we used close to close issues (as if they were not planned), rather than fix or resolve them
It would be good to support that IMO